dwave/system/composites/linear_ancilla.py

```diff
@@ -0,0 +1,213 @@
+# coding: utf-8
+# Copyright 2018 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""Embedding composite to implement linear fields as polarized ancilla qubits.
+"""
+
+import numbers
+
+from typing import Sequence, Mapping, Any
+
+from collections import defaultdict
+
+import numpy as np
+
+from dimod.decorators import nonblocking_sample_method
+import dimod
+
+
+__all__ = ["LinearAncillaComposite"]
+
+
+class LinearAncillaComposite(dimod.ComposedSampler, dimod.Structured):
```

Comment:
```suggestion
class LinearAsFluxBiasAncillaComposite(dimod.ComposedSampler, dimod.Structured):
```

Something a bit more descriptive?

Reply:
I prefer to keep it simple, not sure that Flux Bias adds any necessary information in the name